README.md

```diff
@@ -3,35 +3,37 @@
 [![Greenkeeper badge](https://badges.greenkeeper.io/camacho/format-package.svg)](https://greenkeeper.io/)
 
 <!-- AUTO-GENERATED-CONTENT:START (INSTALL:flags=["-D"]) -->
+
```

Comment:
I'm not sure what's going on with the formatting :( We'll probably want to sort this out in a separate PR. I'll file an issue for it.

Reply:
Done. See #54